### PR TITLE
chore(curriculum): convert survey form into lab

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by zairahira — APPROVED** [review]
I was able to complete the project with using the provided stories 👍️

File: curriculum/challenges/english/25-front-end-development/lab-survey-form/66a8e9ca371965b6ef5f2989.md [hunk 1]
```diff
@@ -0,0 +1,600 @@
+---
+id: 66a8e9ca371965b6ef5f2989
+title: Build a Survey Form
+challengeType: 14
+dashedName: build-a-survey-form
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/lab-survey-form/66a8e9ca371965b6ef5f2989.md:5]:
```suggestion
dashedName: lab-build-a-survey-form
```

**Review by zairahira — COMMENTED** [review]
Would be OK without this too, but this would be good for consistency.

File: curriculum/challenges/english/25-front-end-development/lab-survey-form/66a8e9ca371965b6ef5f2989.md (hunk 1 above)
Comment (moT01) [reply curriculum/challenges/english/25-front-end-development/lab-survey-form/66a8e9ca371965b6ef5f2989.md:5]: Looks like the challenges just stick with the `build-a-survey-form` structure.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**